rules/linux/execution_shell_via_udp_cli_utility_linux.toml

```diff
@@ -71,22 +71,20 @@ timestamp_override = "event.ingested"
 type = "eql"
 query = '''
 sample by host.id, process.pid, process.parent.pid
```

Comment:
pid because auditd doesnt sent entity_id, right?

Reply:
are these not deterministic to use `sequence` instead of `sample`? Is it always process-> socket -> network?

Reply:
`pid because auditd doesnt sent entity_id, right?`

Correct.

`are these not deterministic to use sequence instead of sample? Is it always process-> socket -> network?`

They are deterministic in reality, but not in the stack, as they occur in the same ms. I tried several times, and the sequence differs in each case.